tox.ini

```diff
@@ -232,6 +232,15 @@ commands =
     coverage report --include 'certbot-nginx/*' --show-missing --fail-under=74
 passenv = DOCKER_*
 
+[testenv:integration-certbot-windows]
+commands =
+    {[base]pip_install} acme . certbot-ci
+    pytest certbot-ci/certbot_integration_tests/certbot_tests \
+        --acme-server=pebble \
+        --cov=acme --cov=certbot --cov-report= \
+        --cov-config=certbot-ci/certbot_integration_tests/.coveragerc
+    coverage report --include 'certbot/*' --show-missing --fail-under=64
```

Comment:
Yes it is because of that. It is even 62% on Linux. I update to `integration-certbot`, and add `ACME_SERVER` as for the other integration tests

Reply:
I think it may be nice to also make the minimum coverage configurable through an environment variable which is set to 64 on Windows, but this may be overkill. I'll leave it up to you whether or not to do this.

Reply:
Ultimately, the nginx integration tests will run also on windows, so this specific toxenv will disappear. At this point, it may be interesting to make specific code coverage threshold for each platform, like the unit tests. But for now, I think we can live it like this, since this toxenv is run only for Windows.